docs/docs-components/SkipToContent.js

```diff
@@ -0,0 +1,55 @@
+// @flow strict
+import { useState, useCallback, useEffect, type Node } from 'react';
+import { Box, Text, TapArea } from 'gestalt';
+import { SKIP_TO_CONTENT_ZINDEX } from './z-indices.js';
+
+/**
+ * Skip to content allows you to skip directly to the content on the page
+ *
+ * Note: This TapArea will only show when people are tabbing through (for accessibility purposes)
+ */
+export default function SkipToContent(): Node {
+  const [focused, setFocused] = useState(false);
+  const [mainContent, setMainContent] = useState(null);
+
+  const handleTabIndex = useCallback(() => {
+    mainContent?.removeAttribute('tabindex');
+    mainContent?.removeEventListener('blur', handleTabIndex);
+    mainContent?.removeEventListener('focusout', handleTabIndex);
+  }, [mainContent]);
+
+  useEffect(() => {
+    if (!mainContent) setMainContent(document.querySelector('[role="main"]'));
+
+    return () => {
+      // remove event listener in case we unmount component without calling handleTabIndex
+      mainContent?.removeEventListener('blur', handleTabIndex);
+      mainContent?.removeEventListener('focusout', handleTabIndex);
+    };
+  }, [mainContent, handleTabIndex]);
+
+  return (
+    <TapArea
+      onBlur={() => setFocused(false)}
+      onFocus={() => setFocused(true)}
+      onTap={() => {
+        mainContent?.setAttribute('tabindex', '-1');
+        mainContent?.focus({ preventScroll: true });
+        mainContent?.addEventListener('blur', handleTabIndex);
+        mainContent?.addEventListener('focusout', handleTabIndex);
+      }}
+    >
+      {focused ? (
+        <Box position="fixed" width="100%" zIndex={SKIP_TO_CONTENT_ZINDEX}>
+          <Box width="100%" color="primary" padding={4}>
```

Comment:
Can we try color="education"? Primary is a bit glaring on the site.

Reply:
DONE